lib/BackendClient.js

```diff
@@ -433,6 +439,37 @@ var BackendClient = (function(){
         return deferred.promise;
     }
 
+    function sendBeacon(entryId, clientPartnerId, data, beaconTag) {
+        let tag = beaconPrefix + '_' + beaconTag;
+        let deferred = Q.defer();
+        logger.debug(`[${entryId}] Calling beacon.add, Tag: [${tag}]`);
+
+        let KalturaBeacon = new kalturaVO.KalturaBeacon();
+        KalturaBeacon.relatedObjectType = kalturaTypes.BeaconObjectTypes.ENTRY_SERVER_NODE_BEACON;
+        KalturaBeacon.eventType = tag;
+        KalturaBeacon.objectId = entryId;
+        KalturaBeacon.privateData = JSON.stringify(data);
```

Comment:
it should be        KalturaBeacon.privateData = data
(we are not sending a string, but an object)